src/sentry/api/helpers/group_index/delete.py

```diff
@@ -140,7 +140,7 @@ def create_audit_entries(
         )
 
 
-def delete_groups(
```

Comment:
There's a lot of functions that include this string, thus, making it hard to search through the codebase.